ngrinder-frontend/src/js/components/operation/ScriptConsole.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+    <div class="container">
+        <fieldset>
+            <legend class="header">
+                <span v-text="i18n('navigator.dropDown.scriptConsole')"></span>
+                <button id="run_btn" class="btn btn-success pull-right"
+                        v-text="i18n('operation.script.runScript')" v-on:click="runScript">
+                </button>
+            </legend>
+        </fieldset>
+        <codemirror ref="editor"
+                    :options="cmOptions"></codemirror>
+        <pre class="prettyprint pre-scrollable validation" v-text="result"></pre>
+    </div>
+</template>
+
+<script>
+    import Component from 'vue-class-component';
+    import Base from '../Base.vue';
+
+    import { codemirror } from 'vue-codemirror';
+
+    import 'codemirror/mode/groovy/groovy.js';
+
+    import 'codemirror/addon/display/fullscreen.js';
+    import 'codemirror/addon/dialog/dialog.js';
+    import 'codemirror/addon/search/search.js';
+    import 'codemirror/addon/selection/active-line.js';
+
+    @Component({
+        name: "scriptConsole",
+        components: { codemirror }
+    })
+    export default class ScriptConsole extends Base {
+
+        result = 'You can write groovy code to monitor the ngrinder internal state.\n' +
+            '\n' +
+            'Following variables are available.\n' +
+            '\n' +
+            '- applicationContext (org.springframework.context.ApplicationContext)\n' +
+            '- agentManager (org.ngrinder.perftest.service.AgentManager)\n' +
+            '- agentManagerService (org.ngrinder.agent.service.AgentManagerService)\n' +
+            '- regionService (org.ngrinder.region.service.RegionService)\n' +
+            '- consoleManager (org.ngrinder.perftest.service.ConsoleManager)\n' +
+            '- userService (org.ngrinder.user.service.UserService)\n' +
+            '- perfTestService  (org.ngrinder.perftest.service.PerfTestService)\n' +
+            '- tagService (org.ngrinder.perftest.service.TagService)\n' +
+            '- fileEntryService\t(org.ngrinder.script.service.FileEntryService)\n' +
+            '- config (org.ngrinder.infra.config.Config)\n' +
+            '- pluginManager (org.ngrinder.infra.plugin.PluginManager)\n' +
+            '- cacheManager (org.springframework.cache.CacheManager)\n' +
+            '\n' +
+            'Please type following and click the Submit button as a example\n' +
+            '\n' +
+            'print agentManager.getAllAttachedAgents()\n' +
+            '\n' +
+            'please refer nGrinder javadoc to find out more APIs on the given variables.\n';
+
+        cmOptions = {
```

Comment:
Is there a reason to use computed in below code? 

https://github.com/naver/ngrinder/blob/445efe356db413d4e6c625fc2c3f36ad63607e62/ngrinder-frontend/src/js/components/common/CodeMirror.vue#L43